lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -501,7 +501,7 @@ html.video-fullscreen {
         }
       }
 
-      .vert > .xblock-student_view {
+      .vert {
```

Comment:
Note: this change moves the border below the "staff debug info" button found in the courseware.